src/components/TabsPanels.tsx

```diff
@@ -0,0 +1,36 @@
+interface TabsPanelsProps {
+  index: number;
+}
+
+const TabsPanels = ({ index }: TabsPanelsProps) => {
+  return (
+    <div className="tds-background-grey-50 tds-u-p2">
+      <div hidden={index !== 0}>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ea optio
+        reiciendis suscipit porro veniam? Neque asperiores aut consequuntur
+        delectus ullam facilis nihil rerum, beatae doloribus officiis? Sit
+        tempora sapiente ducimus iste recusandae delectus quos ratione totam
+        earum beatae, voluptate officiis obcaecati! Excepturi corrupti sequi
+        neque iure ex obcaecati, non laboriosam.
+      </div>
+      <div hidden={index !== 1}>
```

Comment:
I think these conditional rendering demonstrates how you could actually put all pages in the same page, but that is a code smell which i don't think we need to document.

Better just have a single div in this case IMO